src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 20px;
+  background-color: #000;
+}
+
+.box {
+  max-width: 350px;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  padding: 10px;
+  background-color: #308dc0;
+  border: 1px solid #e75551;
+}
+
+.box .box {
+  background-color: #ffa500;
+}
+
+.box .box .box {
```

Comment:
We need to go deeper!!!

create unique classes instead of using complex selectors